cli/flox/src/commands/activate.rs

```diff
@@ -371,6 +372,10 @@ impl Activate {
         cmd.arg("--registry");
         cmd.arg(reg_path);
 
+        // Redirect the output streams so watchdog output doesn't appear in the shell
+        cmd.stdout(Stdio::null());
+        cmd.stderr(Stdio::null());
```

Comment:
(praise) I've seen some "unexpected parent" output from the watchdog in failing integration tests recently which this should fix 👍🏻